recipes/somoclu/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "somoclu" %}
+{% set version = "1.7.4" %}
+{% set sha256 = "428d38ee2313fe10703c92b94c36fcab45361eaba2d9881a45bb91ac9b7a26e7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - cuda.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy >=1.10
```

Comment:
If this links to numpy you may want to use `numpy 1.10.*` here, the run section is fine though. The logic is to build with the oldest `numpy` possible (`1.10` here) so you can run with any `numpy` that is `>=`. The current configuration would give you `numpy 1.13.*` for both build and run, breaking this package if you try to run it with `numpy <1.13`.